### PR TITLE
Fix/push

### PR DESCRIPTION
**Pull request by lianxmfor**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -97,6 +97,8 @@ jobs:
           - { online: sqlite,   offline: sqlite,   metadata: sqlite   }
           - { online: mysql,    offline: mysql,    metadata: mysql    }
           - { online: postgres, offline: postgres, metadata: postgres }
+          - { online: redis, offline: postgres, metadata: postgres }
+          - { online: redis, offline: postgres, metadata: postgres }
```
Comment (jinghancc) [inline .github/workflows/ci.yml:101]: hmm is this a typo? seems you added two same combinations

Comment (lianxmfor) [reply .github/workflows/ci.yml]: good catch! fixed.

**Review by wfxr — APPROVED** [review]
LGTM

**Review by jinghancc — APPROVED** [review]
LGTM

**Pull request merged**